kolibri/plugins/learn/assets/src/views/content-page/index.vue

```diff
@@ -20,7 +20,7 @@
       :channelId="channelId"
       :available="content.available"
       :extraFields="content.extra_fields">
-      <icon-button @click="nextContentClicked" v-if="progress >= 1 && showNextBtn" class="next-btn" :text="$tr('nextContent')">
+      <icon-button @click="nextContentClicked" v-if="progress >= 1 && showNextBtn" class="next-btn" :text="$tr('nextContent')" alignment="right">
```

Comment:
Outside of the strict remit of this PR, but could you amend it so that the show next button is shown for all content types, and not just for exercises? Could essentially eliminate the `showNextBtn` computed property.